SPECS/xxhash/xxhash.spec

```diff
@@ -0,0 +1,186 @@
+Version:  0.8.1
+Release:  3%{?dist}
+Summary:  Extremely fast hash algorithm
+#    The source for the library (xxhash.c and xxhash.h) is BSD
+#    The source for the command line tool (xxhsum.c) is GPLv2+
+License:  BSD and GPLv2+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:      http://www.xxhash.com/
+Source0:  https://github.com/Cyan4973/xxHash/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+#    Fix compilation on RHEL 7 ppc64le (gcc 4.8)
+#    https://github.com/Cyan4973/xxHash/issues/622
+#    https://github.com/Cyan4973/xxHash/pull/631
+Patch0:   %{name}-epel7-ppc64le.patch
+
+BuildRequires:	make
+BuildRequires:	gcc
+BuildRequires:	doxygen
+
+%description
+xxHash is an Extremely fast Hash algorithm, running at RAM speed
+limits. It successfully completes the SMHasher test suite which
+evaluates collision, dispersion and randomness qualities of hash
+functions. Code is highly portable, and hashes are identical on all
+platforms (little / big endian).
+
+%package libs
+Summary:	Extremely fast hash algorithm - library
+License:	BSD
+
+%description libs
+xxHash is an Extremely fast Hash algorithm, running at RAM speed
+limits. It successfully completes the SMHasher test suite which
+evaluates collision, dispersion and randomness qualities of hash
+functions. Code is highly portable, and hashes are identical on all
+platforms (little / big endian).
+
+%package devel
+Summary:  Extremely fast hash algorithm - development files
+License:  BSD
+Requires:  %{name}-libs%{?_isa} = %{version}-%{release}
+# By setting XXH_INLINE_ALL, xxhash may be used as a header-only library.
+# Dependent packages that use xxhash this way must BR this virtual Provide:
+Provides:  %{name}-static = %{version}-%{release}
+
+%description devel
+Development files for the xxhash library
+
+%package doc
+Summary:  Extremely fast hash algorithm - documentation files
+License:  BSD
+BuildArch:  noarch
+
+%description doc
+Documentation files for the xxhash library
+
+%prep
+%autosetup -p1 -n xxHash-%{version}
+
+%build
+# Enable runtime detection of sse2/avx2/avx512 on intel architectures
+%ifarch %{ix86} x86_64
+%global dispatch 1
+%else
+%global dispatch 0
+%endif
+
+%make_build MOREFLAGS="%{__global_cflags} %{?__global_ldflags}" \
+        DISPATCH=%{dispatch}
+doxygen
+
+%install
+%make_install PREFIX=%{_prefix} LIBDIR=%{_libdir}
+rm %{buildroot}/%{_libdir}/libxxhash.a
+
+%check
+make check
+make test-xxhsum-c
+
+%ldconfig_scriptlets libs
+
+%files
+%{_bindir}/xxh*sum
+%{_mandir}/man1/xxh*sum.1*
+%license cli/COPYING
+%doc cli/README.md
+
+%files libs
+%{_libdir}/libxxhash.so.*
+%license LICENSE
+%doc README.md
+
+%files devel
+%{_includedir}/xxhash.h
+%{_includedir}/xxh3.h
+%{_libdir}/libxxhash.so
+%{_libdir}/pkgconfig/libxxhash.pc
+
+%files doc
+%doc doxygen/html
+
+%changelog
+* Wed Jul 6 2022 Sumedh Sharma <sumsharma@microsoft.com> - 0.8.1-3
+- Initial import for CBL-Mariner from Fedora 36 (License: BSD & GPLv2+)
```

Comment:
This doesn't seem right- this line should have the license of the spec file itself, not the license of the package. Fedora specs are usually licensed under the MIT license.

Reply:
Yes, this is a mistake. The License field should be for SPEC file rather than sources.
Corrected the changelog.